package.json

```diff
@@ -26,15 +25,18 @@
   },
   "homepage": "https://github.com/fastify/fastify-basic-auth#readme",
   "devDependencies": {
-    "@types/node": "^12.7.12",
-    "fastify": "^2.0.0",
-    "fastify-auth": "^0.3.0",
-    "standard": "^11.0.1",
-    "tap": "^12.0.1"
+    "fastify": "^3.0.0-alpha.1",
+    "fastify-auth": "^0.7.2",
```

Comment:
This needs to be published and then it can be updated with the new types @mcollina is fastify-auth ready to go? I can publish major if so